### DISCUSSION TITLE
now/examples, is there room for a with-webpack example?

### DISCUSSION THREAD

**Discussion by Bakon** [vercel/vercel#4044] [General] [answered]
Hi, yesterday was my first time working with Now, and I kinda like it so far.
But I couldn't find an easy example trying to upload a React app with Webpack onto the Now platform.
I would like to add a very basic example to the examples folders, using Webpack & Webpack-dev-server.

Since Webpack isn't a framework, I'm not sure how you guys feel about adding this as an example.
I do think it would be a nice addition for the less experienced people like myself :)

**Comment by mcsdevv** [comment, marked as answer]
Hey @Julicolo, thanks for raising this. We're going to be reworking examples so that rather than initial framework creation, they feature full examples that you can take and run with. Therefore, it's unlikely we'll have anything webpack related in there.

How did you get on with deployment?

**Comment by Bakon** [comment]
My Webpack config was caching the bundle.js in memory so I had to make it output a bundle for production so that I could deploy it with Now.
It was a very minor tweak so all went well, had a really smooth experience, thanks for asking :)

Thanks for taking the time to explain the upcoming plans with the examples!
